tests/microos/journal_check.pm

```diff
@@ -16,61 +16,67 @@ use warnings;
 use testapi;
 use microos;
 use version_utils 'is_opensuse';
+use Mojo::JSON qw(decode_json);
+
+sub parse_bug_refs {
+    my $res     = shift;
+    my $product = get_required_var('DISTRI');
+    my %bp;
+
+    my $bugs = decode_json($res->body);
+    foreach my $bugid (keys %$bugs) {
+        if (ref $bugs->{$bugid}->{products} eq ref []) {
```

Comment:
IMO it could just mandate that products always has to be an array

Reply:
Yup, make sense. I will update it